spec/api-review-state.spec.ts

```diff
@@ -170,7 +170,7 @@ describe('api review', () => {
       data: [
         {
           user: { id: 1, login: 'ckerr' },
-          body: 'Fix this please!',
+          body: 'API CHANGES REQUESTED',
           state: 'CHANGES_REQUESTED',
```

Comment:
You can use the Request Changes feature in GitHub, but it will only have an impact on API review if you also include `API CHANGES REQUESTED` in your comment.